Makefile

```diff
@@ -1,26 +1,38 @@
+MAKEFLAGS += --warn-undefined-variables
+
 GOCMD = go
 GOMOD = $(GOCMD) mod
 GOBUILD = $(GOCMD) build
 GOCLEAN = $(GOCMD) clean
 GOTEST = $(GOCMD) test
+GOINSTALL = $(GOCMD) install
 CMDDIR = cmd
 DISTDIR = dist
 TARGET = terrajux
 MAIN = $(CMDDIR)/$(TARGET)/main.go
 SOURCES = $(wildcard **/*.go)
 
-.PHONY: test clean tidy release snapshot
+.PHONY: tools clean tidy test snapshot release
+
+all: tools clean tidy test build
 
 $(DISTDIR)/$(TARGET): test $(SOURCES)
-	$(GOBUILD) -o $(DISTDIR)/$(TARGET) -v $(MAIN)
+	$(GOBUILD) -o $(DISTDIR)/$(TARGET) $(MAIN)
+
+tools:
+	$(GOINSTALL) honnef.co/go/tools/cmd/staticcheck@latest
+	$(GOINSTALL) github.com/goreleaser/goreleaser@latest
```

Comment:
feels like there might be a better way to handle installing these test reqs with a blank import and a `// +build tools` tag but i couldn't quite grok how to make that work and install the CLIs in `GOBIN`.